pkg/controller/controller_broker_test.go

```diff
@@ -185,6 +188,9 @@ func TestReconcileBrokerExistingServiceClass(t *testing.T) {
 }
 
 func TestReconcileBrokerExistingServiceClassDifferentExternalID(t *testing.T) {
```

Comment:
This is the kind of description I was looking for:

```go
// Simulate catalog refresh where broker detects an existing service class but
// the service has an ID change.
```

It's not really necessary to declare that we're checking actions and events, since every test like this should be doing that.

I'd like the description above to be in the form of a godoc comment for the test method.